app/views/groups/group_links/_invite_group_modal.html.erb

```diff
@@ -0,0 +1,88 @@
+<%= turbo_frame_tag "new_member_modal" do %>
+  <%= viral_dialog(open: open, classes: ["overflow-visible"]) do |dialog| %>
+    <%= dialog.with_header(title: I18n.t("groups.group_links.new.title")) %>
+    <%= dialog.with_section do %>
+      <%= turbo_frame_tag("invite-group-alert") %>
+      <div class="my-6">
```

Comment:
**nitpick:** This `my-6` is adding 24px to the top and bottom of this paragraph, which is just a little excessive.  From the Flowbite docs, try adding something like `mb-6 text-lg font-normal text-gray-500 lg:text-xl sm:px-16 xl:px-48 dark:text-gray-400`, and play with the sizing.  You should not need a margin on the top as the contain div already has sufficient padding.

Reply:
Fixed in 014f9e9b4a5570cb1de2a0b5602f383091e8b64d